src/client/styles/styles.scss

```diff
@@ -249,3 +249,7 @@ $two-thirds: $one-third * 2;
     background-color: govuk-colour("red");
   }
 }
+
+.grey-text {
+  color:  #868D91;
```

Comment:
minor, but we should use a colour from the palette. it's more likely to be wcag friendly https://design-system.service.gov.uk/styles/colour/#colour-palette